### PR TITLE
feat: adds error boundaries

### LINKED ISSUE #142: Implement Error Boundaries
Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://fb.me/react-error-boundaries to learn more about error boundaries.

### PR DESCRIPTION
**Pull request by farinalvaro**
Closes #142 

https://user-images.githubusercontent.com/8449567/107568253-7c1df700-6bc5-11eb-89b3-1345b3d05747.mov

### CODE REVIEW

File: src/components/pages/UncaughtError.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import React, { FC } from 'react'
+import lostAstronaut from 'assets/images/lostAstronaut.png'
+import GridRow from 'components/atoms/GridRow'
+import RoundBtn from 'components/atoms/RoundBtn'
+
+const UncaughtError: FC = () => (
+  <CenteredPageTemplate
+    title="Oops! Something went wrong."
+    subtitle={
+      `You thought this mission to the moon would be a quick six month thing.
```
Comment (julianmrodri) [inline src/components/pages/UncaughtError.tsx:11]: What are these messages? Provided by default? Should we setup our own message I guess

File: README.md [hunk 2]
```diff
@@ -72,3 +72,7 @@ There are some ways you can make this module better:
 ## License
 
 [MIT](./LICENSE)
+
+## Acknowledgements
+
+- Our `Uncaught Error` page was inspired by [Kapwing](https://www.kapwing.com/404-illustrations). Check out their great job!
```
Comment (jurajpiar) [inline README.md:78]:
```suggestion
- Our `Uncaught Error` page was inspired by [Kapwing](https://www.kapwing.com/404-illustrations). Check out their great work!
```

**Review by jurajpiar — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (farinalvaro) [reply README.md:78]: Haha thank you for this!

File: src/components/pages/UncaughtError.tsx (hunk 1 above)
Comment (farinalvaro) [reply src/components/pages/UncaughtError.tsx]: 👍  totally. Just added more generic message

**Review by julianmrodri — APPROVED** [review]
LGTM

File: src/components/pages/UncaughtError.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    subtitle="We are sorry but something went wrong."
+  >
+    <GridRow justify="center">
+      <RoundBtn onClick={(): void => {
+        window.location.href = window.location.origin
+      }}
+      >
+        Go to Home Page
+      </RoundBtn>
+      <img src={lostAstronaut} alt="lost astronaut" />
```
[the remark this answers is not in the document]
Comment (farinalvaro) [reply src/components/pages/UncaughtError.tsx:19]: Yes, totally agree with that and sorry for too informal message.
Just to clarify, before merging, the text was changed to "We are sorry but something went wrong".

**Pull request merged**